Formula/cayley.rb

```diff
@@ -16,12 +16,18 @@ class Cayley < Formula
   depends_on "mercurial" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULES"] = "on"
 
     dir = buildpath/"src/github.com/cayleygraph/cayley"
     dir.install buildpath.children
 
     cd dir do
+      # Install dependencies
+      system "go", "mod", "download"
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things

Reply:
Will wrap with `resource`

Reply:
Eventually no need for go mod download.